Dependency issues

Two errors occur when running the [example](https://github.com/edukisto/false-language/tree/a073ef6dfdb5e2b072431a6dfc763d8a8a3f0640#examples):

```shell
Error: Cannot find module '/.../node_modules/false-language/main.js'. Please verify that the package.json has a valid "main" entry
```

and

```shell
Error: Cannot find module 'readline-sync'
```

--- a/package.json
+++ b/package.json
@@ -2,13 +2,16 @@
   "name": "false-language",
   "version": "1.0.1",
   "description": "Node.js FALSE interpreter",
-  "main": "main.js",
+  "main": "fls.js",
   "scripts": {
     "test": "node test.js"
   },
   "keywords": ["false", "language", "programming-language"],
   "author": "Illia Oleksiienko <iliiliilya123@gmail.com>",
   "license": "GNU GPL v3.0",
+  "dependencies": {
+    "readline-sync": "0.2.4 - 1.4"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/iliiliiliili/false-language.git"

